test/unit.js

```diff
@@ -78,6 +79,13 @@ function checkScopeSerialization(js, serialization, { earlyErrors = true, asScri
   assert.equal(serialize(globalScope), serialization);
 }
 
+function checkScopeAnnotation(source, { skipUnambiguous = true, skipScopes = true } = {}) {
```

Comment:
Why `skipScopes = true`?

Reply:
Printing annotations on scopes makes the output substantially more noisy and for tests which are just trying to check that variables are resolved correctly, there's no point.